Eat_and_Greet/src/comps/ChooseImageButton/ChooseImageButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, TouchableOpacity, View, StyleSheet } from 'react-native';
+import React from 'react';
+
+interface ChooseImageProps {
+  onPress?: () => void;
```

Comment:
wouldn't this be mandatory? since the point of the button is to select an image

Reply:
Made it optional so we could reuse this for the profile page and have clicking it do nothing if edit isn't active. Also didn't want to write that functionality.